packages/bot/src/listeners/guild/member/guildMemberAddSendSurvey.js

```diff
@@ -26,11 +28,13 @@ class GuildMemberAddSendSurvey extends Listener {
     try {
       const joinedUser = await client.users.fetch(member.id);
       logger.debug(`Sending new user message to: ${joinedUser.username}`);
-      await joinedUser.send({ embeds: [this.buildSurveyMessage(joinedUser)] });
+      await joinedUser.send({ embeds: [this.buildSurveyMessage(joinedUser, $api)] });
```

Comment:
imo we should pull the api data first and pass it as a param to build survery message as opposed to doing the api call inside of it. It will make it more reusable.